HISTORY.md

```diff
@@ -7,6 +7,8 @@
 * Fix a race in DumpStats() with column family destruction due to not taking a Ref on each entry while iterating the ColumnFamilySet.
 * Fix a race in item ref counting in LRUCache when promoting an item from the SecondaryCache.
 * Fix a race in BackupEngine if RateLimiter is reconfigured during concurrent Restore operations.
+* Fix a bug that backup_rate_limit and restore_rate_limit in BackupEngine only apply to writes but not including reads.
```

Comment:
Good thoughts! My description is more about the case where user expects to limit both read and write by passing a rate limiter that can do that but finds out that only write is limited. But now reading your suggestion makes me realize yours is a more precise description for the bug since (1) in the scenario I described, read is somehow also limited by the symmetric property of read and write in Backup engine (2) yours is a more direct and easy-to-understand scenario. 

Will update that according to your suggestion - thanks! 👍